templates/.pgpass.erb

```diff
@@ -0,0 +1 @@
+127.0.0.1:*:pgbouncer:pgbouncer:<%= @pgbouncer_password -%>
```

Comment:
Can you please provide this an an epp template?
From our [guidelines](https://voxpupuli.org/docs/#reviewing-a-module-pr);
> Is a new template added? The preferred language is epp, not erb

Please provide the needed variable as [parameter](https://puppet.com/docs/puppet/5.5/lang_template_epp.html#accessing-variables).

Reply:
sure, will do.